### DISCUSSION TITLE
Calling `octokit.rest.<api>` vs `octokit.<api>`?

### DISCUSSION THREAD

**Discussion by billyvg** [octokit/octokit.js#2178] [Q&A]
I couldn't find anything about this but I noticed that the docs now all reference calling the GH API with `octokit.rest`, is the old method going to be deprecated? Are there any differences?

**Comment by wolfy1339** [comment]
`octokit.js` is a wrapper around all the different `@octokit` modules, including `@octokit/rest`, hence the `octokit.rest` prefix

You can continue to use whichever method you want.
